docs/queries/grouping.md

```diff
@@ -70,6 +70,7 @@ Task properties:
     * Note that the text displayed is generated programmatically and standardised, and so may not exactly match the text in any manually typed tasks. For example, a task with `🔁 every Sunday` is grouped in `every week on Sunday`.
 1. `tags`
     * The tags of the tasks or `(No tags)`. If the task has multiple tags, it will show up under every tag.
+    * Note that tasks with subtags on their global filter will be grouped separately.
```

Comment:
Separately from what? Will the global filter be included in the group name, or removed from it?

Reply:
Separately from each other (this has always been the behavior - works the same on v1.10.0):
<img width="450" alt="image" src="https://user-images.githubusercontent.com/8586879/180624958-9a143db9-a409-4346-ba97-6b24bbd95f05.png">

<img width="450" alt="image" src="https://user-images.githubusercontent.com/8586879/180624975-020c9ec3-67d2-43a6-8549-10efc23c51f0.png">